core/templates/dev/head/app.constants.ts

```diff
@@ -124,6 +124,9 @@ export class AppConstants {
   public static ISSUE_TYPE_MULTIPLE_INCORRECT_SUBMISSIONS =
     'MultipleIncorrectSubmissions';
   public static ISSUE_TYPE_CYCLIC_STATE_TRANSITIONS = 'CyclicStateTransitions';
+
+  public static UNRELATED_ACTIONS_PER_BLOCK_THRESHOLD = 4;
```

Comment:
Add a comment explaining what this means (what is "unrelated"? what is a "block"?) (Note that these are in the general constants file so some notes about this being related to lesson playthroughs might be useful.)

Reply:
Done. Actually, renamed the constant for better understanding.